File: PHP/PROGRAM/Clothing.php
<?php
class Clothing extends Product{
    private $size;      //deklarasi objek - objek class clothing
    private $material;
    private $gender;

    //konstruktor
    public function __construct($idProduct, $name, $brand, $price, $size, $material, $gender)
    {
        parent::__construct($idProduct, $name, $brand, $price); //mewariskan objek - objek dari orang tuanya (class product)
        $this->size = $size;
        $this->material = $material;
        $this->gender = $gender;
    }

    //getter dan setter
    public function get_size(){
        return $this->size;
    }

    public function set_size($size){
        $this->size = $size;
    }

    public function get_material(){
        return $this->material;
    }

    public function set_material($material){
        $this->material = $material;
    }

    public function get_gender(){
        return $this->gender;
    }

    public function set_gender($gender){
        $this->gender = $gender;
    }
}

?>

File: PYTHON/PROGRAM/Clothing.py
from Product import Product     #MENGAMBIL KELAS ORANG TUA AGAR BISA DIWARISKAN

class Clothing(Product):    #PENGISIAN KELAS CLOTHING YANG DIWARISI OLEH PRODUCT
    #KONTRUKTOR
    def __init__(self, idProduct, name, brand, price, size, material, gender):
        super().__init__(idProduct, name, brand, price) # OBJEK WARISAN DARI CLASS ORTU
        self.size = size
        self.material = material
        self.gender = gender
    
    #GETTER DAN SETTER
    def get_size(self):
        return self.size
    
    def set_size(self, size):
        self.size = size
        
    def get_material(self):
        return self.material
    
    def set_material(self, material):
        self.material = material
        
    def get_gender(self):
        return self.gender
    
    def set_gender(self, gender):
        self.gender = gender